backend/LexBoxApi/Services/PermissionService.cs

```diff
@@ -29,22 +48,35 @@ public bool CanSyncProject(Guid projectId)
         return User.Projects.Any(p => p.ProjectId == projectId);
     }
 
+    public async ValueTask<bool> CanSyncProjectAsync(Guid projectId)
+    {
+        if (CanSyncProject(projectId)) return true;
+        // Org managers can sync any project owned by their org(s)
+        return await ManagesOrgThatOwnsProject(projectId);
+    }
+
     public async ValueTask AssertCanSyncProject(string projectCode)
     {
         if (!await CanSyncProject(projectCode)) throw new UnauthorizedAccessException();
     }
 
-    public void AssertCanSyncProject(Guid projectId)
+    public async ValueTask AssertCanSyncProject(Guid projectId)
     {
-        if (!CanSyncProject(projectId)) throw new UnauthorizedAccessException();
+        if (!await CanSyncProjectAsync(projectId)) throw new UnauthorizedAccessException();
     }
 
     public async ValueTask<bool> CanViewProject(Guid projectId)
     {
         if (User is not null && User.Role == UserRole.admin) return true;
         if (User is not null && User.Projects.Any(p => p.ProjectId == projectId)) return true;
-        var project = await dbContext.Projects.FindAsync(projectId);
+        var project = await dbContext.Projects.Include(p => p.Organizations).Where(p => p.Id == projectId).FirstOrDefaultAsync();
```

Comment:
this should also be cached per my previous comment

Reply:
Done in commit 44bc6599.